<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sargan DEMO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbSevSULogo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>293</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/png/sevsu.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbCopyright">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>750</y>
      <width>202</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>© 2023 СевГУ. Все права защищены.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCamera">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>CAMERA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edIPCamera">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>640</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>http://localhost:8080/sargan</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edIP">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>610</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edPort">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3310</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>610</y>
      <width>451</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>УСТАНОВИТЬ СОЕДИНЕНИЕ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbArrowLeft">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>66</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/png/arrow-left.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbArrowRight">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>20</y>
      <width>66</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/png/arrow-right.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbTarget">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/png/target.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edAngleLeft">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edAngleRight">
    <property name="geometry">
     <rect>
      <x>598</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lblCamera</zorder>
   <zorder>lbSevSULogo</zorder>
   <zorder>lbCopyright</zorder>
   <zorder>edIPCamera</zorder>
   <zorder>edIP</zorder>
   <zorder>edPort</zorder>
   <zorder>btnConnect</zorder>
   <zorder>lbArrowLeft</zorder>
   <zorder>lbArrowRight</zorder>
   <zorder>lbTarget</zorder>
   <zorder>edAngleLeft</zorder>
   <zorder>edAngleRight</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
